api/src/services/listing.service.ts

```diff
@@ -1207,22 +1246,25 @@ export class ListingService implements OnModuleInit {
             : undefined,
           listingEvents: dto.listingEvents
             ? {
-                create: dto.listingEvents.map((event) => ({
-                  type: event.type,
-                  startDate: event.startDate,
-                  startTime: event.startTime,
-                  endTime: event.endTime,
-                  url: event.url,
-                  note: event.note,
-                  label: event.label,
-                  assets: event.assets
-                    ? {
-                        create: {
-                          ...event.assets,
-                        },
-                      }
-                    : undefined,
-                })),
+                create: dto.listingEvents.map((event) => {
+                  return {
```

Comment:
question: Was the syntax without the return causing problems? Or is this just a preferred format?

Reply:
nah I was trying to figure out what was going on with some console logs and didn't put things back properly